src/protocols/secure_channel/PASESession.cpp

```diff
@@ -787,7 +792,7 @@ CHIP_ERROR PASESession::ValidateReceivedMessage(ExchangeContext * exchange, cons
     else
     {
         mExchangeCtxt = exchange;
-        mExchangeCtxt->SetResponseTimeout(kSpake2p_Response_Timeout + mExchangeCtxt->GetSessionHandle()->GetAckTimeout());
+        mExchangeCtxt->UseSuggestedResponseTimeout(kExpectedProcessingTime);
```

Comment:
Again, I think this ends up using the wrong timeouts for messages that are not PBKDFParamRequest and PBKDFParamResponse.